protocol/avdl/keybase1/teams.avdl

```diff
@@ -440,6 +440,8 @@ protocol teams {
   TeamID lookupImplicitTeam(string name, boolean public);
   TeamID lookupOrCreateImplicitTeam(string name, boolean public);
 
+  void teamReAddMemberAfterReset(int sessionID, string name, string username);
```

Comment:
The frontend shouldn't have to use `__keybase_implicit_team` names. Hopefully they will have the teamID, so I think this should be `teamReaddMemberAfterReset(int sessionID, TeamID teamID, string username)`